package/run.sh

```diff
@@ -156,7 +156,7 @@ if [ "$CATTLE_CLUSTER" != "true" ]; then
 fi
 
 if [ -z "$CATTLE_NODE_NAME" ]; then
-    CATTLE_NODE_NAME=$(hostname -s)
+    CATTLE_NODE_NAME=$HOSTNAME
```

Comment:
Is this guaranteed to be the "short" hostname, only the portion before the first dot.

Reply:
Here the output:

mvarlese@marco-pc-lavoro:~> hostname -s
marco-pc-lavoro

mvarlese@marco-pc-lavoro:~> hostname
marco-pc-lavoro.fritz.box

Reply:
the man page says `HOSTNAME  Automatically set to the name of the current host`, so it is ambiguous. I tested it within a k8s cluster and outside and it seems to behave equivalent. I'll try to get hostname(1) embedded in the SLES base image as well, so we can change this. here's my testing:

```
$ hostname -f
postgresql-0.hub-1621432880-postgresql-headless.default.svc.cluster.local
$ hostname -s
postgresql-0
$ echo $HOSTNAME
postgresql-0
$ uname -n
postgresql-0

another one:

$ hostname -f
wotan.suse.de
$ hostname -s
wotan
$ echo $HOSTNAME
wotan
$ uname -n
wotan
```

Reply:
I see this affects documented behavior: 

https://github.com/rancher/docs/blame/master/content/rancher/v2.x/en/cluster-provisioning/rke-clusters/custom-nodes/agent-options/_index.md#L20

so I'll have to work harder on getting hostname(1) into the image it seems..

Reply:
The "hostname" executable is now available via the BCI repository (not installed in the base image by default yet). 
You could revert the latest patch and add the package hostname where needed.

Reply:
done.